docs/msource.md

```diff
@@ -0,0 +1,119 @@
+# MSource
+
+-----------
+
+## Point
+
+### Template structure
+
+### Description
+
+### Parameters
+
+### Dependencies
+
+### Products
+
+
+-----------
+
+## SeqConserved
+
+### Template structure
+
+One template argument `FImpl`, expected to be a fermion implementation.
+
+### Description
+
+Sequential source with an insertion of a conserved current $J_\mu$ using a `PropagatorField` $q(x)$ for $t_A\leq x_3 \leq t_B$:
+
+$$source(x) = \sum\limits_{\mu=\mu_{min}}^{\mu_{max}} \theta(x_3 - t_A)\theta(t_B - x_3)\exp(i x\cdot p)\,J_\mu q(x)$$
```

Comment:
If I remember well there are some subtleties about how the split-point part is shifted. A note about that would probably be useful.